vllm/v1/spec_decode/ngram_proposer.py

```diff
@@ -58,13 +59,29 @@ def propose(
               followed that pattern. Here we will return [4,2,3] because 
               we only have three tokens after the match.
         """
+        min_ngram = self.min_n
+        max_ngram = self.max_n
+        k = self.k
+        if vllm_xargs is not None:
```

Comment:
is this supposed to be request level ngram parameters? I couldnt find this in OAI so this is additional?